txlib/api/resources.py

```diff
@@ -31,12 +31,38 @@ def retrieve_content(self):
         self._populated_fields['content'] = res['content']
         return res['content']
 
+    def _create(self, **kwargs):
+        """Create a resource in the remote Transifex server."""
+        path = self._construct_path_to_collection()
+        content = kwargs['content']
+        is_binary = Resource.is_binary(content)
```

Comment:
Based on my comment about the `is_binary()` method, I suggest this gets changed to:
```
is_binary = not isinstance(content, str)
```